subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -586,8 +586,13 @@ static int cmd_wifi_ps(const struct shell *sh, size_t argc, char *argv[])
 		shell_fprintf(sh, SHELL_NORMAL, "PS wake up mode: %s\n",
 				config.ps_params.wakeup_mode ? "Listen interval" : "DTIM");
 
-		shell_fprintf(sh, SHELL_NORMAL, "PS timeout: %d ms\n",
-				config.ps_params.timeout_ms);
+		if (!config.ps_params.timeout_ms) {
```

Comment:
Isn't the logic inverted here? Now it prints the timeout is disabled for values other than 0.